app/abilities/curriculum-inventory.js

```diff
@@ -0,0 +1,12 @@
+import Ember from 'ember';
```

Comment:
I'm not seeing this in use - did I just miss it?  Looks like you are directly using the canViewCurr stuff on the currentUser service which is a much cleaner approach.  I would like to remove ember-can entirely at some point in the near future.